about.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Please record a video presenting the features of the project. Add a link to the video in your pull request description as demanded by the project requirements.
  - The maximum length of the video is 5 minutes.
  - The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
  - The video contains a demo of the project, showing the pages in different screen sizes.
  - The video contains a highlight about a piece of the code you find interesting or you are proud of making.